index.html

```diff
@@ -0,0 +1,179 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>BEIRUT GLOBAL TECHNOLOGY CONFERENCE</title>
+    <link rel="stylesheet" href="style.css">
+    <script src="https://kit.fontawesome.com/4f02d055e5.js" crossorigin="anonymous"></script>
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>  
+    <link href="https://fonts.googleapis.com/css2?family=Lato&display=swap" rel="stylesheet">
+    <link href="http://fonts.cdnfonts.com/css/cocogoose" rel="stylesheet">
+  </head>
+  <body>
+      <header>
+          <nav class="header-nav">
+            <i id="menu" class="fas fa-bars fa-3x burger-menu menu"></i>
+            <div class="mobile-menu">
+              <i class="fas fa-window-close close-item"></i>              
+              <div class="menu-div">
+                <a href="about.html" class="about-menu">About</a>
+                <a href="#program" class="porgram-menu">Program</a>
+                <a href="#speak" class="speak-menu">Speakers</a>
+                <a href="#partners" class="partners-menu">Partners</a>
+                <a href="#footer" class="contact-menu">Contact</a>
+                <a href="/" class="conference-menu">Conference</a>
+              </div>
+            </div>
+            <div class="header-first-line">
+              <ul class="header-list1">
+                <li><a href="/"><i class="fab fa-facebook-f"></i></a></li>
+                <li><a href="/"><i class="fab fa-twitter"></i></a></li>
+                <li><a href="/">English</a></li>
+                <li><a href="/">My Page</a></li>
+                <li><a href="/">Logout</a></li>
+              </ul> 
+            </div>
+            <div class="header-second-line">
+              <div class="header-logo">
+                <a class="header-logo-a" href="index.html">
+                  <i class="fab fa-confluence fa-2x"></i>
+                  <h2>Conference</h2>
+                </a>
+              </div>
+                <ul class="header-list2">
+                  <li><a href="about.html">About</a></li>
+                  <li><a href="#program" >Program</a></li>
+                  <li><a href="#speak" >Speakers</a></li>
+                  <li><a href="#partners" >Partners</a></li>
+                  <li><a href="#footer" >Contact</a></li>
+                  <li ><a href="/" id="head-list-conf" class="header-list-conference">Conference</a></li>
+                </ul> 
+            </div>
+          </nav>
+          <div class="header-content">
```

Comment:
- [ ] OPTIONAL
- The heading of your hero page has text with transparent blend. This feature is not implemented in your design.
- From the **Original design** below we can see the the heading text in the hero section is aligned to the left side same as the logo, as informed by the previous code reviewers.
-  The desktop version of your  header has a dark background which is apparently different from the original design below:


**Your design:**
![image](https://user-images.githubusercontent.com/12745474/128378521-a3a4997f-13f9-477c-a00b-2bea3b108401.png)


**The original design:**

![image](https://user-images.githubusercontent.com/12745474/128377312-7c8a2af1-afd0-4427-91a6-e8f295ee5482.png)


Let's endeavor to improve on the above highlighted issues 😉

Reply:
A,right thank you